package.json

```diff
@@ -2,7 +2,7 @@
     "name": "@apple/hierarchical-confusion-matrix",
     "author": "Apple Inc.",
     "version": "1.0.1",
-    "license": "Apple Sample Code",
+    "license": "SEE LICENSE IN `LICENSE`",
```

Comment:
What's the reason for this change?

Reply:
This gets rid of the warning about incorrect SPDX license during `yarn install`.

Reply:
TIL.